src/components/PlayerInfo.vue

```diff
@@ -8,15 +8,28 @@
       <span v-if="round<6">Income: {{player.income.replace(/,/g, ', ')}}<br/></span>
       Range: {{data.range}}, Terraforming cost: {{3 - data.terraformCostDiscount}}o<br/>
       <span v-if="faction === 'Ivits'">Fed value: {{player.fedValue }}, No fed value: {{player.structureValue - player.fedValue }} <br/></span> 
-
-      <span style="white-space: nowrap; line-height: 1em">
+ 
+        <span style="white-space: nowrap; line-height: 1em">
         Steps: 
         <span v-for="i in [0, 1, 2, 3]" :key="i" :class="{'ml-2': i > 0}">
-          <i v-for="planet in planetsWithSteps(i)" :class="['planet', planet]" :key="planet" /> {{i}}
+          <span v-for="planet in planetsWithSteps(i)"  :key="planet"  >
+            <svg width="15" height="20" viewbox="0 0 15 15" >
+              <circle :cx="8" :cy="8" :r="6"  :class="['player-token', 'planet-fill', planet]" />
+            </svg>
+          </span> 
+          <span>{{i}}</span>
```

Comment:
Maybe add a space just before `{{i}}`, to add a bit of space between the token and the number?

Also, the same comment applies for colonized planets.

Reply:
A space before is adding to much space. Can Work for steps but not for colonized planets, because you are unsure to which one the number is referring to

Reply:
I've added a pixel to the svg container
<img width="312" alt="screen shot 2018-09-28 at 14 00 24" src="https://user-images.githubusercontent.com/29941924/46207235-f0261000-c326-11e8-944a-1937368c8a34.png">